### ISSUE TITLE
Mirrored masters package breaks diglot & triglot packages

### ISSUE THREAD

**Issue by alerque** [sile-typesetter/sile#1748]
As can be seen in the examples on the SILE website, use of the triglot class blows up after the first page because the required frame order is not properly mirrored and some of the typesetter properties are not preserved in the mirrored version.

![20230328_22h21m07s_grim](https://user-images.githubusercontent.com/173595/228345079-4bab2746-d04c-4f38-951d-dd7c8920facd.png)

**Comment by Omikhleia** [comment]
Possibly related to #1386 ?

**Comment by alerque** [comment]
Quite possibly. I haven't had a chance to dig further yet.

**Comment by alerque** [comment]
No, having looked at the frame situation triglot doesn't have either of the conditions suggested in that issue.
